### ISSUE TITLE
Controller::setOpenApi() TypeError (array vs. string)

### ISSUE THREAD

**Issue by Vlczech** [apitte/core#101]
```
TypeError
Argument 1 passed to Apitte\Core\Schema\Builder\Controller\Controller::setOpenApi() must be of the type array, string given, called in P:\xampp\htdocs\vian_sensingapi\protected\vendor\apitte\core\src\DI\Loader\DoctrineAnnotationLoader.php on line 157
```

```
File: ...\Schema\Builder\Controller\Controller.php:149
146:        /**
147:         * @param mixed[] $openApi
148:         */
149:        public function setOpenApi(array $openApi): void
150:        {
151:            $this->openApi = $openApi;
152:        }
153:    
```

Commit 4bf882f709c871c1233dc6344114e0c04a583a4f was the last corect.

**Comment by mabar** [comment]
`@OpenApi` annotations were not working at all before https://github.com/apitte/core/commit/57ff0f8cca5d2bcd51a00c2e012875a3dda3d1ce

Should be fixed by https://github.com/apitte/core/commit/94985d9edabe3e88b1bf27031ffe35da84e712dc, try it please

**Comment by Vlczech** [comment]
It seemed, that it works for me with 4bf882f, but I didn't test it in depth.

Your fix 94985d9 fixed the previous error for me.
